src/service/firebase.js

```diff
@@ -1,5 +1,6 @@
 import firebase from 'firebase/app'
 import 'firebase/auth'
+import 'firebase/firestore'
 
 const firebaseConfig = {
   apiKey: "AIzaSyA5DCG7mVl_l0qMukK3vZV2bhFwfJySNxQ",
```

Comment:
What is the plan to remove this env keys from the code? .env file?

Reply:
We have a task in the backlog for this, also for rotate the credentials